### PR TITLE
Translate 1 file to ko, playground - Checking Unions with Index Signatures

### PR DESCRIPTION
**Pull request by yahma25**
en
* [Checking Unions with Index Signatures.ts](https://github.com/microsoft/TypeScript-Website/blob/v2/packages//playground-examples/copy/en/3-8/Breaking%20Changes/Checking%20Unions%20with%20Index%20Signatures.ts)

번역 리뷰 요청합니다: @yeonjuan @Bumkeyy 
ref #6

감사합니다 😄

### CODE REVIEW

**Review by bumkeyy — COMMENTED** [review]
LGTM

File: docs/playground/ko/3-8/Breaking Changes/Checking Unions with Index Signatures.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+type IdentifierCache = { [key: string]: number };
+
+// 'file_a'가 문자열 값을 가지고 있어서
+// 실패를 의미합니다
+
+const cacheWithString: IdentifierCache = { file_a: "12343" };
+
+// 그러나, 유니언에 넣을 때는
+// 유효성 검사가 실행되지 않을 것입니다:
```
Comment (bumkeyy) [inline docs/playground/ko/3-8/Breaking Changes/Checking Unions with Index Signatures.ts:19]:
```suggestion
// 유효성 검사가 실행되지 않았습니다:
```
[제안]

File: docs/playground/ko/3-8/Breaking Changes/Checking Unions with Index Signatures.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+let userCache: IdentifierCache | { index: number };
+userCache = { file_one: 5, file_two: "abc" };
+
+// 이 부분은 고쳐졌고,
+// 컴파일러에서 'file_two'에 관한 오류가 발생할 것입니다.
```
Comment (bumkeyy) [inline docs/playground/ko/3-8/Breaking Changes/Checking Unions with Index Signatures.ts:25]:
```suggestion
// 컴파일러에서 'file_two'에 관한 오류가 발생합니다.
```
[제안]

**Comment by yahma25** [comment]
@bumkeyy 
오랜만에 놀러 온 기분이네요. 꼼꼼한 체크 감사합니다. 😄

**Comment by bumkeyy** [comment]
LGTM

**Comment by bumkeyy** [comment]
LGTM

**Comment by bumkeyy** [comment]
LGTM

**Comment by bumkeyy** [comment]
> There was an issue merging, maybe try again bumkeyy. [Details](https://github.com/microsoft/TypeScript-Website-Localizations/runs/1656240389?check_suite_focus=true)

@orta 

Errors occur while merging.
If you have time, could you please check it?

**Comment by yeonjuan** [comment]
LGTM

**Comment by orta** [comment]
LGTM

**Comment by orta** [comment]
I'm not wild that https://github.com/microsoft/TypeScript-Website-Localizations/pull/132#issuecomment-1012155945 worked but this doesn't, still not figured out a difference

**Pull request merged**